dags/snowflake_cost_attribution.py

```diff
@@ -36,7 +37,7 @@ def post_metrics(token: str, category: str, type: str, data: list) -> None:
         raise Exception(f"Failed to post data: {resp.text}")
 
 
-@task
+@task(retries=0)
```

Comment:
IMO the env var exists or it doesn't when the pipeline starts; no real need to retry.